extension/js/common/platform/store/key-store.ts

```diff
@@ -43,10 +43,10 @@ export class KeyStore extends AbstractStore {
     return withPp;
   }
 
-  public static add = async (acctEmail: string, newKeyArmored: string) => {
+  public static add = async (acctEmail: string, newKey: string | Key) => {
```

Comment:
Now I see what you meant. Yup, no problem here. Best way to highlight this kind of question is to do a self-review on the PR code, where you add a comment on a particular line, then you click `review changes` on top and use `Comment` type of review. Then I'll know exactly what you are referring to.

If I read the code properly before reading your comment, I would have definitely put the two together, so it's also my own fault. But commenting on a particular line would have made it easier for me to understand the question.